exchangelib/account.py

```diff
@@ -126,6 +126,9 @@ def listen_for_notifications(self, subscription_id, timeout_s=30*60):
     def unsubscribe_from_notifications(self, subscription_id):
         return Unsubscribe(self, [subscription_id]).call()
 
+    def sync_folder_items(self, folders, shape, sync_state=None, ignore=None, max_changes=100):
+        return SyncFolderItems(account=self, folders=folders).call(shape, sync_state, ignore, max_changes)
```

Comment:
I believe SyncFolderItems asserts it only gets 1 Folder, so you might have to change it so it doesn't. Or we could just do one SyncFolderItems for each Folder (which seems easier tbh).